tests/h/services/group_test.py

```diff
@@ -12,82 +12,92 @@
-    def test_create_sets_access_flags_for_group_types(self,
-                                                      service,
-                                                      group_type,
-                                                      flag,
-                                                      expected_value):
-        group = service.create('Anteater fans', 'cazimir', type_=group_type)
+    @pytest.mark.parametrize('flag,expected_value', [
+        ('joinable_by', JoinableBy.authority),
+        ('readable_by', ReadableBy.members),
+        ('writeable_by', WriteableBy.members)])
+    def test_create_private_group_sets_access_flags(self, service, flag, expected_value):
+        group = service.create_private_group('Anteater fans', 'cazimir')
 
         assert getattr(group, flag) == expected_value
 
-    def test_create_raises_for_invalid_group_type(self, service):
-        with pytest.raises(ValueError):
-            service.create('Anteater fans', 'cazimir', type_='foo')
-
-    def test_create_adds_group_to_session(self, db_session, service):
-        group = service.create('Anteater fans', 'cazimir')
+    def test_create_private_group_adds_group_to_session(self, db_session, service):
+        group = service.create_private_group('Anteater fans', 'cazimir')
 
         assert group in db_session
 
-    def test_create_sets_group_ids(self, service):
-        group = service.create('Anteater fans', 'cazimir')
+    def test_create_private_group_sets_group_ids(self, service):
+        group = service.create_private_group('Anteater fans', 'cazimir')
 
         assert group.id
         assert group.pubid
 
-    def test_create_publishes_join_event(self, service, publish):
-        group = service.create('Dishwasher disassemblers', 'theresa')
+    def test_create_private_group_publishes_join_event(self, service, publish):
+        group = service.create_private_group('Dishwasher disassemblers', 'theresa')
 
         publish.assert_called_once_with('group-join', group.pubid, 'acct:theresa@example.com')
 
+    def test_create_open_group_returns_group(self, service, users):
+        creator = users['cazimir']
+
+        group = service.create_open_group(name='test_group',
+                                          userid=creator.username,
+                                          description='test_description')
+
+        assert group.name == 'test_group'
+        assert group.authority == 'example.com'
+        assert group.creator == creator
+        assert group.description == 'test_description'
+        assert group.joinable_by is None
+        assert group.readable_by == ReadableBy.world
+        assert group.writeable_by == WriteableBy.authority
+
+    def test_create_open_group_description_defaults_to_None(self, service):
+        # Create a group with no `description` argument.
+        group = service.create_open_group(name='test_group', userid='cazimir')
+
+        assert group.description is None
```

Comment:
`create_open_group()` really doesn't do anything other than create a `Group` with certain parameters (including looking up the `User` corresponding to the given `userid` and setting it as the group's `creator`, and setting the group's `authority` to `creator.authority`), add the group to the DB, and return the group, so there's very little to test.

Note: These tests don't actually test that the group was saved to the DB! Neither do the `create_private_group()` tests. This is a preexisting issue with the tests so I think it should be fixed in a separate pr.